lib/Backend/amd64/LinearScanMD.cpp

```diff
@@ -574,3 +439,268 @@ RegNum LinearScanMD::GetParamReg(IR::SymOpnd *symOpnd, Func *func)
 
     return reg;
 }
+
+
+IR::Instr* LinearScanMD::GenerateBailInForGeneratorYield(IR::Instr* resumeLabelInstr, BailOutInfo* bailOutInfo)
+{
+    Assert(!bailOutInfo->capturedValues || bailOutInfo->capturedValues->constantValues.Empty());
+    Assert(!bailOutInfo->capturedValues || bailOutInfo->capturedValues->copyPropSyms.Empty());
+    Assert(!bailOutInfo->liveLosslessInt32Syms || bailOutInfo->liveLosslessInt32Syms->IsEmpty());
+    Assert(!bailOutInfo->liveFloat64Syms || bailOutInfo->liveFloat64Syms->IsEmpty());
+    return this->bailIn.GenerateBailIn(resumeLabelInstr, bailOutInfo);
+}
+
+LinearScanMD::GeneratorBailIn::GeneratorBailIn(Func* func, LinearScanMD* linearScanMD):
```

Comment:
looks like this is needed for arm/arm64/x86 still? probably want to get most of this into Lower.cpp and use platform independent lowering